.github/ISSUE_TEMPLATE/Enhancement_issue.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Feature request
+about: Request an improvement or a new feature.
+title: ''
+labels: 'Type: enhancement'
+---
+#### Describe the new feature
+
+<Describe the new feature and how the software should behave once it is implemented.>
+	
+#### Motivation behind the suggestion
+
+<Provide information about the current behaviour and how the new feature fits into CIL.>
+<Optional: provide an example showing the need of the feature.>
+
+	
+#### Optional: proposed solution
+
+<Describe your proposed changes to implement the new feature.>
+<Provide links to relevant information.>
+
```

Comment:
would merge all of this into one short paragraph... or even leave it blank

```suggestion
```

Reply:
At the group meeting it seemed we were happy with this file @paskino Are we?